rlcard/envs/env.py

```diff
@@ -163,6 +163,8 @@ def run(self, is_training=False):
             state = self.get_state(player_id)
             trajectories[player_id].append(state)
 
```

Comment:
I believe we don't want to print trajectories in this method, which will affect all the games.

Reply:
Sure. Shld Set this to WIP! WIll resolve the status once the feature is done